tests/integration/src/install_tests.rs

```diff
@@ -0,0 +1,1009 @@
+        user_groups: vec![custom_user_group_id.clone()],
+    };
+
+    let one_from_custom_user_group = RequestPolicyRuleDTO::Quorum(station_api::QuorumDTO {
+        approvers: station_api::UserSpecifierDTO::Group(vec![custom_user_group_id.clone()]),
+        min_approved: 1,
+    });
+
+    let request_policy_add_user_id = Uuid::new_v4().hyphenated().to_string();
+    let named_rule_dependent_id = Uuid::new_v4().hyphenated().to_string();
+
+    let permissions = vec![
+        InitPermissionInput {
+            resource: ResourceDTO::User(UserResourceActionDTO::List),
+            allow: allow_custom_user_group_users.clone(),
+        },
+        InitPermissionInput {
+            resource: ResourceDTO::User(UserResourceActionDTO::Read(ResourceIdDTO::Any)),
+            allow: allow_custom_user_group_users.clone(),
+        },
+        InitPermissionInput {
+            resource: ResourceDTO::UserGroup(ResourceActionDTO::List),
+            allow: allow_custom_user_group_users.clone(),
+        },
+        InitPermissionInput {
+            resource: ResourceDTO::UserGroup(ResourceActionDTO::Read(ResourceIdDTO::Any)),
+            allow: allow_custom_user_group_users.clone(),
+        },
+        InitPermissionInput {
+            resource: ResourceDTO::Account(AccountResourceActionDTO::List),
+            allow: allow_custom_user_group_users.clone(),
+        },
+        InitPermissionInput {
+            resource: ResourceDTO::Account(AccountResourceActionDTO::Read(ResourceIdDTO::Any)),
+            allow: allow_custom_user_group_users.clone(),
+        },
+        InitPermissionInput {
+            resource: ResourceDTO::Asset(ResourceActionDTO::List),
+            allow: allow_custom_user_group_users.clone(),
+        },
+        InitPermissionInput {
+            resource: ResourceDTO::Asset(ResourceActionDTO::Read(ResourceIdDTO::Any)),
+            allow: allow_custom_user_group_users.clone(),
+        },
+        InitPermissionInput {
+            resource: ResourceDTO::NamedRule(ResourceActionDTO::List),
+            allow: allow_custom_user_group_users.clone(),
+        },
+        InitPermissionInput {
+            resource: ResourceDTO::NamedRule(ResourceActionDTO::Read(ResourceIdDTO::Any)),
+            allow: allow_custom_user_group_users.clone(),
+        },
+        InitPermissionInput {
+            resource: ResourceDTO::RequestPolicy(ResourceActionDTO::List),
+            allow: allow_custom_user_group_users.clone(),
+        },
+        InitPermissionInput {
+            resource: ResourceDTO::RequestPolicy(ResourceActionDTO::Read(ResourceIdDTO::Any)),
+            allow: allow_custom_user_group_users.clone(),
+        },
+        InitPermissionInput {
+            resource: ResourceDTO::Permission(PermissionResourceActionDTO::Read),
+            allow: allow_custom_user_group_users.clone(),
+        },
+    ];
+
+    let request_policies = vec![
+        // edit specific request policy, in the wrong order on purpose
```

Comment:
what does "wrong order" mean? (this test is supposed to succeed)

Reply:
It means the policy is referencing another policy ID, that is specifier later in the list. It is meant to test the dependency-sensitive nature of creating the initial policies.